main.tf

```diff
@@ -0,0 +1,39 @@
+locals {
+  script       = "launch_ranchhand.sh"
+  ip_addresses = join(",", var.node_ips)
+  ansbile_ssh_proxy = var.ssh_proxy_host == "" ? "" : format("-o StrictHostKeyChecking=no -o ProxyCommand=\"ssh -o StrictHostKeyChecking=no -W %%h:%%p -q %s@%s\"", var.ssh_proxy_user, var.ssh_proxy_host)
```

Comment:
`ansbile_ssh_proxy` -> `ansible`

Reply:
Are you suggesting renaming the the variable that is storing the **ssh proxy** args?